content/en/docs/concepts/scheduling/scheduling-framework.md

```diff
@@ -209,88 +206,35 @@ interfaces have the following form.
 
 ```go
 type Plugin interface {
-   Name() string
+    Name() string
 }
 
 type QueueSortPlugin interface {
-   Plugin
-   Less(*v1.pod, *v1.pod) bool
+    Plugin
+    Less(*v1.pod, *v1.pod) bool
 }
 
 type PreFilterPlugin interface {
-   Plugin
-   PreFilter(PluginContext, *v1.pod) error
+    Plugin
+    PreFilter(context.Context, *framework.CycleState, *v1.pod) error
 }
 
 // ...
 ```
 
-# Plugin Configuration
+## Plugin Configuration
 
 Plugins can be enabled in the scheduler configuration. Also, default plugins can
-be disabled in the configuration. In 1.15, there are no default plugins for the
+be disabled in the configuration. Starting from 1.18, there are a number of [default 
+plugins](/docs/reference/scheduling/profiles/#scheduling-plugins) enabled for the
 scheduling framework.
 
-The scheduler configuration can include configuration for plugins as well. Such
-configurations are passed to the plugins at the time the scheduler initializes
-them. The configuration is an arbitrary value. The receiving plugin should
-decode and process the configuration.
-
-The following example shows a scheduler configuration that enables some
-plugins at `reserve` and `preBind` extension points and disables a plugin. It
-also provides a configuration to plugin `foo`.
-
-```yaml
-apiVersion: kubescheduler.config.k8s.io/v1alpha1
-kind: KubeSchedulerConfiguration
-
-...
-
-plugins:
-  reserve:
-    enabled:
-    - name: foo
-    - name: bar
-    disabled:
-    - name: baz
-  preBind:
-    enabled:
-    - name: foo
-    disabled:
-    - name: baz
-
-pluginConfig:
-- name: foo
-  args: >
-    Arbitrary set of args to plugin foo
-```
+In addition to default plugins, you can also implement your own scheduling
+plugins and get them configured along with default plugins. You can visit
+[scheduler-plugins](https://github.com/kubernetes-sigs/scheduler-plugins) for more details.
```

Comment:
nit: consider moving this inside a (new) What's Next section:
```markdown
(end of main text)
{{% /capture %}}
{{% capture whatsnext %}}
(further reading, etc)
{{% /capture %}}

Reply:
I'd put them under this section as they more fit here.